docs/versioned-plugins/codecs/cef-v4.1.3.asciidoc

```diff
@@ -33,22 +33,22 @@ produce an event with the payload as the 'message' field and a '_cefparsefailure
 [cols="<,<,<",options="header",]
 |=======================================================================
 |Setting |Input type|Required
-| <<{version}-plugins-{type}s-{plugin}-delimiter>> |<<string,string>>|No
-| <<{version}-plugins-{type}s-{plugin}-fields>> |<<array,array>>|No
-| <<{version}-plugins-{type}s-{plugin}-name>> |<<string,string>>|No
-| <<{version}-plugins-{type}s-{plugin}-product>> |<<string,string>>|No
-| <<{version}-plugins-{type}s-{plugin}-severity>> |<<string,string>>|No
-| <<{version}-plugins-{type}s-{plugin}-signature>> |<<string,string>>|No
-| <<{version}-plugins-{type}s-{plugin}-vendor>> |<<string,string>>|No
-| <<{version}-plugins-{type}s-{plugin}-version>> |<<string,string>>|No
+| <<{version}-plugins-{type}s-{plugin}-delimiter>> |{logstash-ref}/configuration-file-structure.html#string[string]|No
```

Comment:
The script needs to replace these external links to point to the Logstash Reference.